catalogs/data/airbyte/terraform/azure/outputs.tf

```diff
@@ -0,0 +1,28 @@
+output "connection_string" {
+  value = data.azurerm_storage_account.airbyte.primary_connection_string
```

Comment:
i feel like there's got to be a more secure way to configure this if possible?

Reply:
I should be able to put it in a secret, but we will still need to provide it. Connection string is the only method presented in Airbyte chart values.